### PR TITLE
less logspam "Allele is null"

### PR DESCRIPTION
**Pull request by Fiyorin**

### CODE REVIEW

File: src/main/java/gregtech/common/bees/GT_AlleleHelper.java [hunk 1]
```diff
@@ -141,13 +141,13 @@ public static void initialisation() {
     public <T extends Enum<T> & IChromosomeType> void set(IAllele[] alleles, T chromosomeType, IAllele allele) {
 
         if (allele == null) {
-            GT_Mod.GT_FML_LOGGER.info("Allele is null!");
+            // GT_Mod.GT_FML_LOGGER.info("Allele is null!");
```
Comment (boubou19) [inline src/main/java/gregtech/common/bees/GT_AlleleHelper.java:144]: i'm against this. I'd rather find out why it is null than just nuking the log. That's called hidding things under the carpet and i'm not a fan of it.

File: src/main/java/gregtech/common/bees/GT_AlleleHelper.java [hunk 2]
```diff
@@ -141,13 +141,13 @@ public static void initialisation() {
...
             return;
         }
 
         if (!chromosomeType.getAlleleClass()
             .isInstance(allele)) {
-            GT_Mod.GT_FML_LOGGER.info("chromosomeType is not an instance of allele!" + allele.getName());
+            // GT_Mod.GT_FML_LOGGER.info("chromosomeType is not an instance of allele!" + allele.getName());
```
Comment (boubou19) [inline src/main/java/gregtech/common/bees/GT_AlleleHelper.java:150]: same

**Review by boubou19 — CHANGES_REQUESTED** [review]

File: src/main/java/gregtech/common/bees/GT_AlleleHelper.java (hunk 2 above)
Comment (Fiyorin) [reply src/main/java/gregtech/common/bees/GT_AlleleHelper.java:150]: fair enough: "must go deeper"

Comment (Fiyorin) [reply src/main/java/gregtech/common/bees/GT_AlleleHelper.java:150]: i give up, this code has way too many types to figure out what's going on -> close PR

**Pull request closed without merging**